rds_to_s3_pipeline/etl_pipeline.py

```diff
@@ -126,21 +126,29 @@ def upload_to_s3(parquet_file: str, bucket: str, s3_key: str) -> None:
         raise
 
 
-if __name__ == "__main__":
-    db_connection = get_db_connection()
-    complete_dataframe = load_data_to_dataframe(db_connection)
+def run_pipeline():
+    '''
+    Function that runs the data pipeline from the short term storage (RDS) 
+    to the long term storage (S3)
+    '''
+    connection = get_db_connection()
+    complete_dataframe = load_data_to_dataframe(connection)
 
-    CURRENT_DATE = datetime.now().strftime("%Y-%m-%d")
-    PARQUET_FILE = save_to_parquet(complete_dataframe, CURRENT_DATE)
-    S3_KEY = f"{S3_KEY_PREFIX}{CURRENT_DATE}.parquet"
+    curent_date = datetime.now().strftime("%Y-%m-%d")
+    parquet_file = save_to_parquet(complete_dataframe, curent_date)
+    s3_key = f"{S3_KEY_PREFIX}{curent_date}.parquet"
 
-    if not isinstance(PARQUET_FILE, str):
+    if not isinstance(parquet_file, str):
         raise ValueError(f"""Expected string for local file, got {
-                         type(PARQUET_FILE)}""")
+                         type(parquet_file)}""")
 
-    upload_to_s3(PARQUET_FILE, S3_BUCKET, S3_KEY)
+    upload_to_s3(parquet_file, S3_BUCKET, s3_key)
 
-    if os.path.exists(PARQUET_FILE):
-        os.remove(PARQUET_FILE)
+    if os.path.exists(parquet_file):
+        os.remove(parquet_file)
         logging.info("Temporary Parquet file removed: %s",
-                     PARQUET_FILE)
+                     parquet_file)
+
+
+if __name__ == "__main__":
+    run_pipeline()
```

Comment:
all looks good to me. Is the pylint score okay? I had to make current_date, parquet_file and S3 upper case to increase the pylint score.